src/main/java/gov/nih/nci/evs/api/fhir/R4/FhirUtilityR4.java

```diff
@@ -354,10 +355,56 @@ public static void notSupported(
           format(
               "Input parameter '%s' is not supported%s",
               paramName, (additionalDetail == null ? "." : format(" %s", additionalDetail)));
+      throw exception(message, IssueType.NOTSUPPORTED, 400);
+    }
+  }
+
+  /**
+   * Not supported.
+   *
+   * @param request the request
+   * @param paramName the param name
+   */
+  public static void notSupported(final HttpServletRequest request, final String paramName) {
+    if (request.getParameterMap().containsKey(paramName)) {
+      final String message = format("Input parameter '%s' is not supported", paramName);
       throw exception(message, OperationOutcome.IssueType.NOTSUPPORTED, 400);
     }
   }
 
+  /**
+   * Not supported search params.
+   *
+   * @param request the request
+   */
+  public static void notSupportedSearchParams(final HttpServletRequest request) {
+    for (final String param :
```

Comment:
Nit. This should be defined as a static array field. Like we would with a constant. Otherwise this array would  get created every time this method is called when the value would not change

Reply:
Definitely not a nit. I will update this in R4 and R5 to rely on a static list to search through to avoid unnecessary variable creation